src/coreclr/vm/ilstubresolver.h

```diff
@@ -53,7 +53,6 @@ class ILStubResolver : public DynamicResolver
     void SetStubTargetMethodSig(PCCOR_SIGNATURE pStubTargetMethodSig, DWORD cbStubTargetSigLength);
     void SetStubMethodDesc(MethodDesc* pStubMD);
 
-    COR_ILMETHOD_DECODER* AllocGeneratedIL(size_t cbCode, DWORD cbLocalSig, UINT maxStack);
     COR_ILMETHOD_DECODER* GetILHeader();
     COR_ILMETHOD_SECT_EH* AllocEHSect(size_t nClauses);
```

Comment:
All `SetStub....` methods and `AllocEHSect` can be next to `AllocGeneratedIL` under `DACCESS_COMPILE` too.